src/vs/editor/common/config/editorOptions.ts

```diff
@@ -1196,7 +1196,7 @@ class EditorFind extends BaseEditorOption<EditorOption.find, EditorFindOptions>
 						nls.localize('editor.find.autoFindInSelection.always', 'Always turn on Find in selection automatically'),
 						nls.localize('editor.find.autoFindInSelection.multiline', 'Turn on Find in selection automatically when multiple lines of content are selected.')
 					],
-					description: nls.localize('find.autoFindInSelection', "Controls whether the find operation is carried out on selected text or the entire file in the editor.")
+					description: nls.localize('find.autoFindInSelection', "Controls when the find operation is carried out on selected text instead of the entire file in the editor.")
```

Comment:
I think the old structure is a bit confusing. I think we should update it to say "Controls the condition for turning on find in selection automatically."